sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DeadStores.cs

```diff
@@ -892,6 +892,15 @@ string Local()
                 return buffer;
             }
         }
+    }
 
+    // https://github.com/SonarSource/sonar-dotnet/issues/3304
+    public class Repro_3304
+    {
+        public void WithRefKeyword(int[] values)
+        {
+            ref int value = ref values[0]; // Noncompliant FP because there exists write that is propagated
+            value = default; // Noncompliant FP because it's ref variable and value is propagated somewhere
```

Comment:
```suggestion
            value = default; // Noncompliant FP (see above comment)
```

Reply:
I'd keep the original text, because there are two different FPs with two different reasoning behind them. I took some time to understand the difference so I'd like to help our future ourselves.